lib/notifications/bridge.ex

```diff
@@ -0,0 +1,136 @@
+defmodule Notifications.Bridge do
+  @moduledoc """
+  Queries and maintains the status of a bridge
+  """
+
+  use GenServer
+  require Logger
+
+  defstruct bridge1_status: nil,
+            bridge1_lowering_time: []
+
+  @type t :: %__MODULE__{
+          bridge1_status: String.t() | nil,
+          bridge1_lowering_time: DateTime.t() | nil
+        }
+
+  @fetch_ms 60 * 1_000
+
+  @spec default_name() :: GenServer.name()
+  def default_name(), do: Notifications.Bridge
+
+  @spec start_link() :: GenServer.on_start()
+  @spec start_link(Keyword.t()) :: GenServer.on_start()
```

Comment:
Double typespec (which, interestingly enough, doesn't seem to cause either `mix compile` or `mix dialyzer` to fail? That's weird.).